app/iscsi_tgt/iscsi_tgt.c

```diff
@@ -77,6 +77,7 @@ usage(char *executable_name)
 	printf(" -H         show this usage\n");
 	printf(" -d         disable coredump file enabling\n");
 	printf(" -q         disable notice level logging to stderr\n");
+	printf(" -f         run iscsi target foreground, the default is background\n");
```

Comment:
I think the default should be the other way around to avoid breaking current users - default to not running as a daemon and require a flag to turn it on.